crates/testing/src/test_builder.rs

```diff
@@ -70,6 +71,21 @@ pub struct TestDescription {
     pub unreliable_network: Option<Box<dyn NetworkReliability>>,
     /// view sync check task
     pub view_sync_properties: ViewSyncTaskDescription,
+    /// description of builders to run
+    pub builders: Vec1<BuilderDescription>,
+}
+
+#[derive(Clone, Debug)]
+pub enum BuilderChange {
+    Up,
+    Down,
+    FailClaims(bool),
+}
+
+#[derive(Clone, Debug)]
+pub struct BuilderDescription {
+    /// Views to toggle builder on
+    pub changes: HashMap<u64, BuilderChange>,
```

Comment:
I am not super clear on the name of the variable compared to the doc comment. Is this saying that we have some views where a builder may be on or off? I think perhaps `changes` might not be fully descriptive of what this mechanism does. Furthermore, is this intended to be pre-filled before the test? Or during as a sort of state machine that gets written to and deleted from when done?

Reply:
Yeah, that's very lazy documentation. Thanks for calling me out on that, I'll make it more sensible.
`changes` is prescriptive to the builder, telling it how to behave (stop, start, continue working but fail claim calls), I named it by analogue with "node changes" mechanism in the harness which does the same for HS nodes. Do you have any suggestions on alternative naming?